R/S3.R

```diff
@@ -552,7 +552,7 @@ pander.htest <- function(x, caption = attr(x, 'caption'), ...) {
 
     if (is.null(caption)) {
         if (is.null(storage$caption)) {
-            caption <- paste0(x$method, ': `', gsub('( and | by )', '`\\1`', x$data.name), '`')
+            caption <- paste0(x$method, ': `', gsub('( and | by )', '`\\1`', paste0(x$data.name)), '`')
```

Comment:
I think it should rather be: `paste(x$data.name, collapse = '')`
Can you please update in this PR?